payjoin-cli/tests/e2e.rs

```diff
@@ -83,7 +83,7 @@ mod e2e {
                 .await
                 .expect("Failed to write to stdout");
 
-            if line.starts_with("BITCOIN") {
+            if line.starts_with("BITCOIN") || line.starts_with("bitcoin") {
```

Comment:
not very sure why this is suddenly needed.. the pjuribuilder implementation generate uris with "bitcoin" where previously we had "BITCOIN".. didnt investigate that yet tho

Reply:
https://github.com/Kixunil/bip21/pull/20

I prefer something like this to ignore case
```suggestion
            if line.to_ascii_uppercase().starts_with("BITCOIN") {
```